files/install-deps-worker.yaml

```diff
@@ -44,6 +44,7 @@
           - epel-rpm-macros
           # for the `pkg_tool` switch to allow centpkg
           - centpkg
+          - osh-cli
```

Comment:
Is it packaged for CentOS 9 Stream too?

Reply:
I'll answer myself after checking :D no, but there are builds in Copr `@openscanhub/devel`

Reply:
But I don't see CLI in there :eyes:

```
osh-hub 1.0.0.20240704.103731.g4de668b
osh-common 1.0.0.20240704.103731.g4de668b
osh-worker 1.0.0.20240704.103731.g4de668b
osh-hub-conf-devel 1.0.0.20240704.103731.g4de668b
osh-worker-conf-devel 1.0.0.20240704.103731.g4de668b
osh-client 1.0.0.20240704.103731.g4de668b
```

Reply:
pushed the fix, please have a look

Reply:
thanks for having a look into this, LGTM!